manifests/init.pp

```diff
@@ -132,8 +146,8 @@
   }
 
   #build the databases base piece of the config file
-  concat::fragment { "$dbtmpfile":
-    target  => "$conffile",
+  concat::fragment { $::pgbouncer::dbtmpfile:
```

Comment:
Apply the same style here. No need to use the namespace, since it's referencing a local variable from a parameter